src/components/WPScan/WPScan.tsx

```diff
@@ -1,116 +1,107 @@
-    const [selectedDetectionMode, setSelectedDetectionMode] = useState(""); // State variable to store selected detection mode.
-    const [selectedOutputFormat, setSelectedOutputFormat] = useState(""); // State variable to store selected output format.
-    const [verboseChecked, setVerboseChecked] = useState(false); // State variable to check if verbose mode is enabled.
-    const [stealthyChecked, setStealthyChecked] = useState(false); // State variable to check if verbose mode is enabled.
-    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
-    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
-    const [loadingModal, setLoadingModal] = useState(true); // State variable that indicates if the modal is opened.
-    const [opened, setOpened] = useState(!isCommandAvailable); // State variable to indicate loading state of the modal.
-    const [allowSave, setAllowSave] = useState(false); // State variable to indicate if saving is allowed
-    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if the output has been saved
+function WPScan() {
+    // Declare state variables for component
+    const [loading, setLoading] = useState(false);
+    const [output, setOutput] = useState("");
+    const [pid, setPid] = useState("");
+    const [allowSave, setAllowSave] = useState(false);
+    const [hasSaved, setHasSaved] = useState(false);
+    const [active, setActive] = useState(0);
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false);
+    const [opened, setOpened] = useState(!isCommandAvailable);
+    const [loadingModal, setLoadingModal] = useState(true);
 
-    // Component Constants.
-    const title = "WPScan"; // Title of the component.
-    const description = "WPScan scans remote WordPress installations to find security issues"; // Description of the component.
+    // Additional state variables for section visibility
+    const [basicOpened, setBasicOpened] = useState(true);
+    const [advancedOpened, setAdvancedOpened] = useState(false);
+    const [authOpened, setAuthOpened] = useState(false);
+    const [outputOpened, setOutputOpened] = useState(false);
+
+    // Declare constants for the component
+    const title = "WPScan";
+    const description = "WPScan scans remote WordPress installations to find security issues";
     const steps =
         "Step 1: Enter a WordPress URL.\n" +
-        "Step 2: Enter any additional options for the scan.\n" +
-        "Step 3: Enter any additional parameters for the scan.\n" +
-        "Step 4: Click Scan to commence WPScan's operation.\n" +
-        "Step 5: View the Output block below to view the results of the tool's execution.\n" + //Steps to run the component
+        "Step 2: Enter any additional parameters for the scan.\n" +
+        "Step 3: Click Scan to commence WPScan's operation.\n" +
```

Comment:
The convention is Start + title of tool. 
```suggestion
        "Step 3: Click Start " + title + " to commence WPScan's operation.\n" +
```